<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DelaySettingClass</class>
 <widget class="QWidget" name="DelaySettingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>延时设置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>延时设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QWidget" name="widget" native="true">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>9</y>
           <width>161</width>
           <height>51</height>
          </rect>
         </property>
         <property name="title">
          <string>延时器1</string>
         </property>
         <widget class="QLabel" name="labelDisNumOne">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>22</y>
            <width>141</width>
            <height>21</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>161</width>
           <height>51</height>
          </rect>
         </property>
         <property name="title">
          <string>延时器2</string>
         </property>
         <widget class="QLabel" name="labelDisNumTwo">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>22</y>
            <width>141</width>
            <height>21</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="buttonSwitch">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>30</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>切换</string>
         </property>
        </widget>
        <widget class="QPushButton" name="buttonClear">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>80</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>清除</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="2">
          <widget class="QPushButton" name="buttonThree">
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="buttonTwo">
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="buttonOne">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="buttonSeven">
           <property name="text">
            <string>7</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="buttonEight">
           <property name="text">
            <string>8</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="buttonFour">
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="buttonFive">
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="buttonSix">
           <property name="text">
            <string>6</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="buttonNine">
           <property name="text">
            <string>9</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="buttonZero">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="buttonSure">
           <property name="text">
            <string>确定</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
